probe-rs-tools/Cargo.toml

```diff
@@ -38,6 +38,7 @@ thiserror = { workspace = true }
 tracing = "0.1"
 typed-path = "0.10"
 probe-rs-mi = { workspace = true }
+enum-map = "2.7.3"
```

Comment:
```suggestion
enum-map = "2.7"
```

We typically don't include the patch version :) It will take the latest anyways.